src/main/java/com/minecolonies/items/ItemProxyColonySize.java

```diff
@@ -0,0 +1,24 @@
+package com.minecolonies.items;
+
+/**
+ * This is a multi used proxy class for the size related achievements.
+ * 
+ * @since 0.3
+ */
+public class ItemProxyColonySize extends AbstractItemMinecolonies
+{
+
+    private String name;
+
+    public ItemProxyColonySize(String name)

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)